db/migrate/20150130201130_add_event_type_id_to_position.rb

```diff
@@ -0,0 +1,5 @@
+class AddEventTypeIdToPosition < ActiveRecord::Migration
+  def change
+    add_column :positions, :event_type_id, :integer
```

Comment:
What is the proper way to do this then? I thought you add the model_id column and then add the association on the models and you're good?